package.json

```diff
@@ -16,8 +16,6 @@
         "start:showcase": "yarn initialize && yarn install:dependencies && yarn link:components && cd demos/showcase && yarn start",
         "lint": "cd components && yarn lint && cd ../demos/showcase && yarn lint && cd ../storybook && yarn lint",
         "lint:fix": "cd components && yarn lint:fix && cd ../demos/showcase && yarn lint:fix && cd ../storybook && yarn lint:fix",
-        "prettier": "prettier \"docs/**.{ts,tsx,js,jsx,json,css,scss,md,html}\" --write && cd components && yarn prettier && cd ../demos/showcase && yarn prettier && cd ../storybook && yarn prettier",
```

Comment:
I don't think you meant to remove the entire prettier command here, right? Just the line affecting the docs folder.

Reply:
I still like to run the prettier command from the root since it will do all of the demos and the components in one command.